microcosm_pubsub/registry.py

```diff
@@ -72,7 +93,7 @@ def find(self, media_type):
 
         try:
             # use a concrete schema class if any
-            schema_cls = self._mappings[media_type]
+            schema_cls = self._mappings[matching_media_type]
```

Comment:
Will this work properly with prefixes? I think you are potentialy returning a prefix on line 74?

Reply:
The assumption here is that the media type that's registered in `self._mappings` the library is either the full media type or the wildcard media type (`application/vnd.foo.*`). So when I pass in `applocation/vnd.foo.bar` I should first "translate" it to the registered wildcard, and only the wildcard from there on.